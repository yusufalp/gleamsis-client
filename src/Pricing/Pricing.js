import React from 'react';
import {Link} from 'react-router-dom';

class Pricing extends React.Component {
  render() {
    return (
      <div>
        <header>
          <h2>Pricing</h2>
          <p>
            We have a solution for every budget
          </p>
        </header>
        <main>
          <div className="group4">
            <section className="item-pricing">
              <h3>Individual Schools</h3>
              <p>
                Small size schools
              </p>
              <ul>
                <li>Up to 1000 users</li>
                <li>Up to 10 admin users</li>
                <li>24/7 phone support</li>
                <li>One-Time set up fee</li>
              </ul>
              <button className='btn-price' type="submit"><Link to='/demo'>Demo GleamSIS</Link></button>
            </section>
            <section className="item-pricing">
              <h3>Districts</h3>
              <p>
                Larger or multiple schools
              </p>
              <ul>
                <li>Up to 5000 users</li>
                <li>Up to 100 admin users</li>
                <li>24/7 phone and online chat support</li>
                <li>No set up fee</li>
              </ul>
              <button className='btn-price' type="submit"><Link to='/contact'>Request a qoute</Link></button>
            </section>
            <section className="item-pricing">
              <h3>Enterprise</h3>
              <p>
                Contact us for a personalized offer
              </p>
              <button className='btn-price' type="submit"><Link to='/contact'>Contact</Link></button>
            </section>
          </div>
        </main>
      </div>
    )
  }
}

export default Pricing;