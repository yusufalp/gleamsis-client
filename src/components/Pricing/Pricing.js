import React from 'react';
import {Link} from 'react-router-dom';

class Pricing extends React.Component {
  render() {
    return (
      <div>
        <header>
          <h2>Pricing</h2>
          <p>
            We have a solution for every budget
          </p>
        </header>
        <main>
          <div className='group4'>
            <section className='item-pricing'>
              <h3>Individual Schools</h3>
              <p>
                Small size schools
              </p>
              <ul>
                <li>Up to 1000 users</li>
                <li>Up to 10 admin users</li>
                <li>24/7 phone support</li>
                <li>One-Time set up fee</li>
              </ul>
              <Link to='/demo' className='btn-add'>Demo GleamSIS</Link>
            </section>
            <section className='item-pricing'>
              <h3>Districts</h3>
              <p>
                Larger or multiple schools
              </p>
              <ul>
                <li>Up to 5000 users</li>
                <li>Up to 100 admin users</li>
                <li>24/7 phone and online chat support</li>
                <li>No set up fee</li>
              </ul>
              <Link to='/contact' className='btn-add'>Request a qoute</Link>
            </section>
            <section className='item-pricing'>
              <h3>Enterprise</h3>
              <p>
                Contact us for a personalized offer
              </p>
              <Link to='/contact' className='btn-add'>Contact</Link>
            </section>
          </div>
        </main>
      </div>
    )
  }
}

export default Pricing;