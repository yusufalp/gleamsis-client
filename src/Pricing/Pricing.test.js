import React from 'react';
import ReactDOM from 'react-dom';
import Pricing from './Pricing';
import renderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';

describe('Pricing component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <Pricing />
      </MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Pricing />
        </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})