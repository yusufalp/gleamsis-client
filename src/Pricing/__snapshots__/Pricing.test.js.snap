// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pricing component renders the UI as expected 1`] = `
<div>
  <header>
    <h2>
      Pricing
    </h2>
    <p>
      We have a solution for every budget
    </p>
  </header>
  <main>
    <div
      className="group4"
    >
      <section
        className="item-pricing"
      >
        <h3>
          Individual Schools
        </h3>
        <p>
          Small size schools
        </p>
        <ul>
          <li>
            Up to 1000 users
          </li>
          <li>
            Up to 10 admin users
          </li>
          <li>
            24/7 phone support
          </li>
          <li>
            One-Time set up fee
          </li>
        </ul>
        <button
          className="btn-price"
          type="submit"
        >
          <a
            href="/demo"
            onClick={[Function]}
          >
            Demo GleamSIS
          </a>
        </button>
      </section>
      <section
        className="item-pricing"
      >
        <h3>
          Districts
        </h3>
        <p>
          Larger or multiple schools
        </p>
        <ul>
          <li>
            Up to 5000 users
          </li>
          <li>
            Up to 100 admin users
          </li>
          <li>
            24/7 phone and online chat support
          </li>
          <li>
            No set up fee
          </li>
        </ul>
        <button
          className="btn-price"
          type="submit"
        >
          <a
            href="/contact"
            onClick={[Function]}
          >
            Request a qoute
          </a>
        </button>
      </section>
      <section
        className="item-pricing"
      >
        <h3>
          Enterprise
        </h3>
        <p>
          Contact us for a personalized offer
        </p>
        <button
          className="btn-price"
          type="submit"
        >
          <a
            href="/contact"
            onClick={[Function]}
          >
            Contact
          </a>
        </button>
      </section>
    </div>
  </main>
</div>
`;
